src/app/components/search-box/search-box.component.html

```diff
@@ -0,0 +1,39 @@
+<mat-card>
+  <mat-card-content>
+    <form [formGroup]="searchForm">
+      <mat-form-field>
+        <input type="text" placeholder="City" aria-label="Number" matInput [formControl]="cities"
```

Comment:
Why aren't you using controls from `searchForm`, but instead you made `separated form-control`? Aria label should be `number`. By default, the input is `type="text"`, so there is no need of specifying this in that case.